testing/test_assertion.py

```diff
@@ -687,7 +688,7 @@ def test_attrs_with_attribute_comparison_off(self):
         @attr.s
         class SimpleDataObject:
             field_a = attr.ib()
-            field_b = attr.ib(cmp=False)
+            field_b = attr.ib(**attrs_no_eq)
```

Comment:
Ditto here, although again I think `attrs_no_eq` is easier on the eyes than `**{ATTRS_EQ_FIELD: False}`